independent-publisher-connectors/Proposal - 24 Pull Request (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "24 pull request",
+    "description": "connects to '24 pull request' api for open source projects contibution",
+    "version": "1.0",
+    "contact": {
+      "name": "Bernard",
+      "email": "bkaraba14@gmail.com"
+    }
+  },
+  "host": "24pullrequests.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/users.json": {
+      "get": {
+        "responses": {
```

Comment:
default responses in not allowed, please define a proper response for this action.

Reply:
fixed